### PR TITLE
Tweak title of npm on ipfs preference

### PR DESCRIPTION
**Pull request by andrew**
As requested by @jessicaschilling in https://github.com/ipfs-shipyard/ipfs-desktop/pull/1739#issuecomment-763893741

Screenshot 2021-01-22 at 13 50 36

### CODE REVIEW

**Review by jessicaschilling — APPROVED** [review]
LGTM but I'll defer to @lidel for OK/merging 😊

File: assets/locales/en.json
Comment (lidel) [inline assets/locales/en.json:231]:
```suggestion
    "npmOnIpfs": "Enable npm on IPFS"
```

**Review by lidel — CHANGES_REQUESTED** [review]
Given this is a toggle checkbox which installs/uninstalls, and other Experiments start with _Enable_, can we do the same? :thinking:

File: assets/locales/en.json
Comment (jessicaschilling) [reply assets/locales/en.json:231]: I'd specifically asked to use "Enable" since it's a little more accurate, and avoids visual repetition. If we' use "Enable" for each one, we might as well remove the word "Enable" altogether.

Comment (lidel) [reply assets/locales/en.json:231]: Let's go with Enable here, and after I merge all other PRs we can clean it up later.

**Pull request merged**
